dill/_dill.py

```diff
@@ -23,14 +23,7 @@
 
 __module__ = 'dill'
 
-import logging
-log = logging.getLogger("dill")
-log.addHandler(logging.StreamHandler())
-def _trace(boolean):
-    """print a trace through the stack when pickling; useful for debugging"""
-    if boolean: log.setLevel(logging.INFO)
-    else: log.setLevel(logging.WARN)
-    return
+from .logger import adapter as logger
```

Comment:
How about we add: `from .logger import trace as _trace` to minimize existing code breakage due to this PR?  Otherwise, LGTM.

Reply:
People should be using `dill.detect.trace`.  However... I've found that if it's easy to keep backward compatibility, you should do it.  Here the cost essentially nothing because we are already importing from the dill logging module.